src/pysplunkhec.py

```diff
@@ -0,0 +1,89 @@
+# the following allows JSON to be posted to Splunk HEC (HTTP Event Collector)
+
+import os
+import requests
+import json
+from src.kms import decrypt
+
+class PySplunkHEC:
+    def __init__(
+        self,
+        hec_token,
+        hec_host,
+        index="main",
+        host="null",
+        source="null",
+        sourcetype="json",
+        hec_port='443'
```

Comment:
Will the `sourcetype` and `hec_port` change at any point?

Reply:
I took code from [here](https://github.com/alphagov/cyber-cloudwatch-fluentd-to-hec/) and expanded, idea being I could take the changes in this repo and update the `cyber-cloudwatch-fluentd-to-hec` code.. eventually making the linked thing a library we could import here - I'd have done this already but MVP and all :)